examples/PathHandler.js

```diff
@@ -60,9 +60,10 @@ PathHandler.prototype.addShape = function(shape) {
  *  @param {Number} x
  *  @param {Number} y
  */
-PathHandler.prototype.arcAbs = function(rx, ry, xAxisRotation, largeArcFlag, sweepFlag, x, y) {
+PathHandler.prototype.arcAbs = function(/* rx, ry, xAxisRotation, largeArcFlag, sweepFlag, x, y */) {
     // TODO: implement once we support arcs
     this.lastCommand = "A";
+    throw new Error("Not yet supported!");
```

Comment:
I tend to frown upon throwing exceptions, but in this case, I agree that this is a good idea.